libs/designer/src/lib/core/utils/monitoring/binders/__test__/inputs/default.test.ts

```diff
@@ -17,7 +17,7 @@ describe('DefaultInputsBinder', () => {
 
   it('should return an empty object when inputs are undefined', () => {
```

Comment:
we should probably change the description of these tests "should return undefined when inputs are undefined"

Reply:
ohh sure, I will update them. Thanks!